### PR TITLE
User page

### PR DESCRIPTION
**Pull request by Lansyfonseka**
Created new page for check user progress.
Url for page "http://localhost:3000/id". This url will be change.
I have some problem with navigation of two sliders on one page. Fix is coming soon.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -14,6 +14,7 @@ import {ProfileSettings} from "./features/profile/ProfileSettings/ProfileSetting
 import {MainPage} from "./pages/Main/MainPage";
 import {Authorization} from "./components/Authorization";
 import {CommonButton} from "./components/CommonButton/CommonButton";
+import { UserPage } from './pages/UserPage/UserPage';
```
Comment (AmidaWolf) [inline src/App.tsx:17]: We should use style guide. In this point it's mean same formatting for file:
```
import {CommonButton} from "./components/CommonButton/CommonButton";
import {UserPage} from './pages/UserPage/UserPage';
```
All imports with spaces, or all imports without spaces. This part should be fixed with eslint+prettier and workspace configuration.

File: src/assets/icons/icon_done.tsx [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export const iconDone = <svg width="51" height="42" viewBox="0 0 51 42" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (AmidaWolf) [inline src/assets/icons/icon_done.tsx:1]: I think, better if we try to use width/height/color as props for all svg icons. With these props we can change size or fill color for every svg icon outside. 
Check, please, in another places too.

File: src/pages/UserPage/UserPage.scss [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+  padding: 60px 0;
+  background-image: url('../../assets/img/sectionLastBG.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  .container {    
+    display: flex;
+    gap: 200px;
+  }
+  .UserDescription {
+    color: #fff;
```
Comment (AmidaWolf) [inline src/pages/UserPage/UserPage.scss:18]: Check [style guide](https://codeguide.academy/html-css.html), colors should be with 6 symbols. Also, all colors should be placed to one general file like `colors` or smth.

File: src/pages/UserPage/UserPage.tsx [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+              <li>
+                <div className='UserDescription-shortResults-speciality'>QA Manual</div>
+                <div className='UserDescription-shortResults-level'>Уровень прожарки</div>
+                <div className='UserDescription-shortResults-levelIcon'>{IconUserLevel}</div>
+              </li>
+              <li>
+                <div className='UserDescription-shortResults-speciality'>AQA Pyton</div>
+                <div className='UserDescription-shortResults-level'>Уровень прожарки</div>
+                <div className='UserDescription-shortResults-levelIcon'>{IconUserLevel}</div>
+              </li>
```
Comment (AmidaWolf) [inline src/pages/UserPage/UserPage.tsx:61]: Can we place all this text to one array and use `for`? DRY as is.

File: src/pages/UserPage/UserPage.tsx [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+                    </div>
+                  </SwiperSlide>
+                )
+              })
+              }
+            </Swiper>
+            <button className='SwiperArrow' ref={nextElRef2}>{arrowSlider}</button>
+          </div>
+        </div>
+      </section>
```
Comment (AmidaWolf) [inline src/pages/UserPage/UserPage.tsx:176]: I think, we can move this section to separated component. Perhaps later we will be able to reuse it in other places. If not - it's just for better readability.

File: src/styles/mixins.scss [hunk 6]
```diff
@@ -4,4 +4,11 @@
   font-weight: 400;
   font-size: 14px;
   color: $primary-dark-text;
+}
+
+@mixin container {
```
Comment (AmidaWolf) [inline src/styles/mixins.scss:9]: About container. We can use it more flexibly by giving it the necessary properties in media expressions to support different screen sizes. Just a suggestion, I don't know the design features.

File: src/App.tsx (hunk 1 above)
Comment (Lansyfonseka) [reply src/App.tsx]: Fix this

Comment (AmidaWolf) [reply src/App.tsx]: Problem still here.
изображение
I think, this is something like workspace configuration.

File: src/assets/icons/icon_done.tsx (hunk 2 above)
Comment (Lansyfonseka) [reply src/assets/icons/icon_done.tsx:1]: Fix this maybe later.
I think simple set width in css and icon resize proportional.

File: src/pages/UserPage/UserPage.scss (hunk 3 above)
Comment (Lansyfonseka) [reply src/pages/UserPage/UserPage.scss]: fix this

File: src/pages/UserPage/UserPage.tsx (hunk 5 above)
Comment (Lansyfonseka) [reply src/pages/UserPage/UserPage.tsx]: Separated compenent fix this problem

File: src/pages/UserPage/UserPage.tsx [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+            <img src={UserPhoto} alt="userPhoto" />
+          </div>
+          <div className='UserDescription'>
+            <div className='UserDescription-name'>Алена Павлова</div>
+            <ul className='UserDescription-shortResults'>
+              {[1,2,3].map(el => {
+                return <li key={el}>
+                <div className='UserDescription-shortResults-speciality'>QA Manual</div>
+                <div className='UserDescription-shortResults-level'>Уровень прожарки</div>
+                <div className='UserDescription-shortResults-levelIcon'>{IconUserLevel}</div>
```
Comment (AmidaWolf) [inline src/pages/UserPage/UserPage.tsx:38]: I mean, we can create something like this:
```
const levelText = 'Уровень прожарки';

const userDescription = [
  {
    speciality: 'QA Manual',
    levelText,
    icon: IconUserLevel,
  },
  {
    speciality: 'QA Manual',
    levelText,
    icon: IconUserLevel,
  },
  {
    speciality: 'AQA Pyton',
    levelText,
    icon: IconUserLevel,
  },
];

const UserDescriptions = () => {
  return (
    <ul>
      {userDescription.map((user, index) => (
        <li key={index}>
          <div className='UserDescription-shortResults-speciality'>{user.speciality}</div>
          <div className='UserDescription-shortResults-level'>{user.levelText}</div>
          <div className='UserDescription-shortResults-levelIcon'>{user.icon}</div>
        </li>
      ))}
    </ul>
  );
};

export default UserDescriptions;

```

Comment (Lansyfonseka) [reply src/pages/UserPage/UserPage.tsx:38]: I suggest this data received from backend. Now it is just example

File: src/App.tsx (hunk 1 above)
Comment (Lansyfonseka) [reply src/App.tsx]: I think fix this after install pretier and es-lint

File: src/pages/UserPage/UserPage.tsx (hunk 7 above)
Comment (AmidaWolf) [reply src/pages/UserPage/UserPage.tsx:38]: Yes, this data will come from the backend and most likely in exactly the form in which I stored it. Therefore, such a structure is completely justified. However, it's up to you.

**Review by AmidaWolf — APPROVED** [review]

**Review by iraskiba — APPROVED** [review]

**Pull request merged**